### DISCUSSION TITLE
Enhancement: Repeat current song in freeplay

### DISCUSSION THREAD

**Discussion by ray1997** [FunkinCrew/Funkin#1652] [Ideas]
It woukd be a little nice addition to toggle repeat current song either on freeplay menu or pause menu.

**Comment by ghost** [comment]
I think it would be a nice addition to the pause menu

**Comment by ghost** [comment]
Yeah

**Comment by ray1997** [comment]
> how would this work? do you just mean once its completed it restarts?

Exactly!
